.github/workflows/test.yml

```
@@ -19,109 +19,192 @@ jobs:
-          platform: ${{ 'iOS Simulator' }}
+          scheme: 'DashSync-Example'
+          platform: 'iOS Simulator'
         run: |
-          xcodebuild test-without-building -scheme "DashSync-Example" -workspace "DashSync.xcworkspace"  -destination "platform=$platform,name=iPhone 13" -testPlan LockTests
+          xcodebuild test-without-building \
+            -scheme "$scheme" \
+            -workspace "DashSync.xcworkspace" \
+            -destination "platform=$platform,name=iPhone 13" \
+            -testPlan LockTests \
+            CODE_SIGN_STYLE=Manual \
+            CODE_SIGN_IDENTITY="Sign to Run Locally"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)